cpp/src/io/utilities/column_buffer.hpp

```diff
@@ -57,7 +57,6 @@ inline rmm::device_buffer create_data(data_type type,
   std::size_t data_size = size_of(type) * size;
 
   rmm::device_buffer data(data_size, stream, mr);
-  CUDF_CUDA_TRY(cudaMemsetAsync(data.data(), 0, data_size, stream.value()));
```

Comment:
I don't think we can just remove this.  Other formats besides parquet use `column_buffer`. It should be controlled via a param that only parquet sets to false.